### ISSUE TITLE
Binary Crossentropy Loss function and Binary Accuracy Metric missing

### ISSUE THREAD

**Issue by Andrea-Oliveri** [deephealthproject/eddl#107]
Good Afternoon,
I looked at the documentation and at the list of Deep Learning features and I did not find the binary_crossentropy loss function as well as the binary_accuracy metric.
I was hence wondering wether you planned on releasing it in a nearby future nonetheless and, if not, how could I implement them with the current status of eddl. 
Thank you in advance for your help

**Comment by RParedesPalacios** [comment]
Both are easy to implement and will be in the future. When? not sure but I guess that deadline would be June. Perhaps you can try with the cross_entropy and accuracy just implemented using two neurons on the output instead of one.
